### PR TITLE
Added CHANNEL_ID variable (needs to be set to a value) in config.env.…

### PR DESCRIPTION
**Pull request by atauln**
…py, and created a webhook Slack integration that posts to CHANNEL_ID when a major project is submitted (major_project_submission.py)

### CODE REVIEW

**Comment by atauln** [comment]
The config file needs to be editted to add a webhook url pointing to #major-projects/whatever channel you'd like.

**Comment by atauln** [comment]
Wait I forgor, the CHANNEL_ID variable should actually be a WEBHOOK_URL variable, because the config var needs a webhood url

File: conditional/blueprints/major_project_submission.py [hunk 1]
```diff
@@ -127,4 +123,4 @@ def major_project_delete(pid, user_dict=None):
     return "Must be project owner to delete!", 401
 
 def send_slack_ping(payload):
-    return requests.post(app.config['CHANNEL_ID'], json.dumps(payload))
\ No newline at end of file
+    return requests.post(app.config['WEBHOOK_URL'], json.dumps(payload))
```
Comment (jabbate19) [inline conditional/blueprints/major_project_submission.py:126]: Newline pls :)

**Review by jabbate19 — CHANGES_REQUESTED** [review]
Nit, and pending testing

File: conditional/blueprints/major_project_submission.py [hunk 2]
```diff
@@ -12,10 +15,16 @@
...
 
+app = Flask(__name__)
+
 logger = structlog.get_logger()
 
 major_project_bp = Blueprint('major_project_bp', __name__)
 
+# Load default configuration and any environment variable overrides (literally copy-pasted from __init.py__)
+_root_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+app.config.from_pyfile(os.path.join(_root_dir, "config.env.py"))
```
[the remark this answers is not in the document]
Comment (jabbate19) [reply conditional/blueprints/major_project_submission.py]: Ignore this was fixed b4 I could post review

File: conditional/blueprints/major_project_submission.py (hunk 1 above)
Comment (atauln) [reply conditional/blueprints/major_project_submission.py]: Is that common practice in coding? Why?

**Review by atauln — COMMENTED** [review]
Should be good?

**Comment by atauln** [comment]
Commits Borked

**Pull request closed without merging**